packages/math-rendering/src/__tests__/render-math.test.js

```diff
@@ -50,10 +52,25 @@ describe.only('render-math', () => {
   it('call render math for an array of elements', () => {
     const divOne = document.createElement('div');
     const divTwo = document.createElement('div');
+
     renderMath([divOne, divTwo]);
+
     expect(MathJax.document).toHaveBeenCalledTimes(1);
     expect(MathJax.findMath).toHaveBeenCalledWith({
       elements: [divOne, divTwo]
     });
   });
+
+  it('wraps the math containing element the right way', () => {
```

Comment:
I would like to see more of the scenarios covered by tests.. there were some old tests in editable html that should probably be brought across: https://github.com/pie-framework/pie-lib/blob/develop/packages/editable-html/src/plugins/math/__tests__/index.test.jsx#L48-L114

Reply:
I thought that I'm going to move those tests after the other PR is merged, will be moving them now.